libs/stack/stack-ui/src/index.ts

```
@@ -62,7 +62,7 @@ export type { TOptionProps } from './components/fields/Option/interface'
 export { Radio, ReactHookFormRadio } from './components/fields/Radio'
 export { default as RadioGroup } from './components/fields/RadioGroup'
 export { default as SearchField } from './components/fields/SearchField'
-export type { default as TSearchProps } from './components/fields/SearchField/interface'
+export type { default as TSearchProps, TSearchActionProps } from './components/fields/SearchField/interface'
```

ESLint

[error] Expected "TSearchActionProps" to come before "TSearchProps".

(perfectionist/sort-named-exports)